plugin/build.gradle.kts

```diff
@@ -0,0 +1,76 @@
+@file:Suppress("UnstableApiUsage")
+
+import com.aliucord.gradle.AliucordExtension
+import com.android.build.gradle.LibraryExtension
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.jlleitschuh.gradle.ktlint.KtlintExtension
+
+// TODO: Change to your repository
+val repo = "Aliucord/plugins-template"
+
+subprojects {
+    val libs = rootProject.libs
+
+    apply {
+        plugin(libs.plugins.android.library.get().pluginId)
+        plugin(libs.plugins.aliucord.get().pluginId)
+        plugin(libs.plugins.kotlin.android.get().pluginId)
+        plugin(libs.plugins.ktlint.get().pluginId)
+    }
+
+    configure<LibraryExtension> {
+        namespace = "com.github.yournamehere"
+
+        compileSdk = 34
+
+        defaultConfig {
+            minSdk = 24
+        }
+
+        buildFeatures {
+            renderScript = false
+            shaders = false
+            buildConfig = true
+            resValues = false
```

Comment:
don't we support compiling layouts/strings?
or am i insane

Reply:
Maybe, i've never used or seen it used though. I disabled cause there was the com.aliucord.plugins.R class generated and it never gets used I think

Reply:
fair
until we have proper resources support I don't think there's a point